tests/unit/sqlalchemy/test_compiler.py

```diff
@@ -29,6 +30,13 @@
     Column('id', Integer, primary_key=True),
     Column('name', String),
 )
+table_with_catalog = Table(
+    'table',
+    metadata,
+    Column('id', Integer, primary_key=True),
```

Comment:
Is there some way to tell SQLAlchemy that our dialect doesn't support `primary_key`? The SQL it generates will lead to syntax errors when actually executed against Trino.

cc: @mdesmet @dungdm93 any ideas?

Reply:
Allowing to define but then silently ignoring them when creating feels misleading. Better to be explicit and fail fast in my opinion. I don't know if that's possible to implement though.

Reply:
Either way it'll probably end up being a separate PR so I won't block this PR for this reason but we should follow up on this.

Reply:
Follow-up issue created: #205

Reply:
Thanks @hashhar @mdesmet ! I didn't realize originally that it was the `PRIMARY KEY` bit that was problematic.